src/plugins/expressions/common/execution/execution.ts

```diff
@@ -420,21 +420,20 @@ export class Execution<
             : of(resolvedArgs);
 
           return args$.pipe(
-            // @ts-expect-error upgrade to ts v4.7.4
-            tap((args) => this.execution.params.debug && Object.assign(head.debug, { args })),
+            tap((args) => this.execution.params.debug && Object.assign(head.debug ?? {}, { args })),
```

Comment:
Nit: for all the cases below, couldn't we instead use spread operator? It seems easier to read.

```suggestion
            tap((args) => this.execution.params.debug && {...head.debug, args}),
```